### PR TITLE
Reduce warnings for Maven 3 builds

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Comment by apmasell** [comment]
This was originally written by Kyle; I just extracted it. Therefore, I am reviewing it and :+1:.

**Comment by przemekk13** [comment]
:+1:

File: notification-server/pom.xml [hunk 1]
```diff
@@ -176,9 +176,15 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (frankie13) [inline notification-server/pom.xml]: :100: :white_check_mark:

**Comment by frankie13** [comment]
:+1:

**Pull request merged**